cmd/tealdbg/util.go

```diff
@@ -118,11 +118,11 @@ func IntToVLQ(v int, buf *bytes.Buffer) {
 }
 
 // MakeSourceMapLine creates source map mapping's line entry
-func MakeSourceMapLine(tcol, sindex, sline, scol int) string {
```

Comment:
this intentionally has a generic interface in order to be used when/if needed

Reply:
I didn't see anyplace else this was used and the caller was doing some weird stuff to get it in the right format. Where else _might_ this eb used?